Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -623,6 +623,33 @@ class RenderNodeTranslatorTests: XCTestCase {
         }
 
     }
+
+    /// Verify that symbols with ellipsis operators don't get curated into an unnamed protocol implementation section.
+    func testAutomaticImplementationsWithExtraDots() throws {
+        let fancyProtocolSGFURL = Bundle.module.url(
+            forResource: "FancyProtocol.symbols", withExtension: "json", subdirectory: "Test Resources")!
+
+        // Create a test bundle copy with the symbol graph from above
+        let (bundleURL, bundle, context) = try testBundleAndContext(copying: "TestBundle", excludingPaths: [], codeListings: [:]) { url in
+            try? FileManager.default.copyItem(at: fancyProtocolSGFURL, to: url.appendingPathComponent("FancyProtocol.symbols.json"))
+        }
+        defer {
+            try? FileManager.default.removeItem(at: bundleURL)
+        }
```

Comment:
Nit: `testBundleAndContext` uses a test teardown block to remove any created temp files.

```suggestion
```

Reply:
Yeah, I've started to clean it up as I encounter them. Here are [some changes from my link resolution PR](https://github.com/apple/swift-docc/pull/251/files#diff-3099e4ce75fa95794446a025b0298bcf588a4fb39d82947ec485b04699d89a23). That PR broke a lot of tests in the early stages of the integration 😅